src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -41,7 +41,8 @@ export default function DateHistogramAggType(timefilter, config, Private) {
     makeLabel: function (agg) {
       const output = this.params.write(agg);
       const params = output.params;
-      return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+      const field = params.field || agg.params.field.displayName || '';
```

Comment:
I'd generally agree, but I'm playing it safe since we're so close to release with limited time to let this bake.